db/migrations/20221115093055_settings_table.sql

```diff
@@ -0,0 +1,131 @@
+-- migrate:up
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name varchar(30) NOT NULL,
+  nickname varchar(30) NOT NULL UNIQUE,
+  password varchar(30) NOT NULL,
```

Comment:
```suggestion
  password varchar(100) NOT NULL,
```

암호화된 비밀번호는 60자로 나올테니 그 이상의 length로 스키마를 정해 주세요!

Reply:
수정하였습니다!!!!

Reply:
아직 푸시가 안된거 같아요~ 확인 부탁드립니다~!